### ISSUE TITLE
🐛 Bug Report: Server Error with document attribute ids containing a dot or hyphen

### ISSUE THREAD

**Issue by tzdev0** [appwrite/appwrite#3033]
### 👟 Reproduction steps

I am trying to add a new document to a fresh collection using the Appwrite Console resulting in a _Server Error_ warning. In the Appwrite container logs the following error wil be thrown:

```bash
[Error] Timestamp: 2022-03-29T08:49:31+00:00
[Error] Method: POST
[Error] URL: /v1/database/collections/:collectionId/documents
[Error] Type: TypeError
[Error] Message: Exception::__construct(): Argument #1 ($message) must be of type string, null given
[Error] File: @swoole-src/library/core/Database/PDOStatementProxy.php
[Error] Line: 64
```

It seems connected to using attributes with an ID containing a hyphen or a dot.

Steps to reproduce:
1. Start with a fresh Appwrite installation.
2. Register with the first user and add a new project with an arbitrary name.
3. Add a new collection with an arbitrary name.
4. Create a new attribute containing an allowed hyphen or dot in the middle of the attribute id.
5. Add a new document to the collection.

### 👍 Expected behavior

A new document will be created containing the attribute with the hyphen/dot id.

### 👎 Actual Behavior

An error message will be shown and no new document will be created.

### 🎲 Appwrite version

Version 0.13.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by Meldiron** [comment]
Interesting 🤔 This issue throws the same error as https://github.com/appwrite/appwrite/issues/3044

Your exact steps really help to dig into the issue, great job on that! 💪 Ill make sure to investigate it tomorrow.

**Comment by Meldiron** [comment]
The issue has been addressed at the library level. Once the new database library version is released, Appwrite will be upgraded to include this bug fix.

**Comment by Meldiron** [comment]
Released with 0.14 💪 Feel free to re-open the issue if the problem still persists for you.
